CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# v97.2.0 (_2023-03-08_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v97.1.0...v97.2.0)
+
+## General
+
+### ⚠️ Breaking Changes ⚠️
+
+- Android: The JVM compatibility target is now version 11 ([#5401](https://github.com/mozilla/application-services/issues/5401))
```

Comment:
Aaah if this is a breaking change, we should release 98.0.0, right?

Unless our consumers are already compatible?

Reply:
If so, lets change this to a "What's changed", and a part of the changelog mention that it's technically a breaking change, but our consumers won't break

Reply:
Great point, I'll reach out and ask the android folks. Looks like that change was by @rvandermeulen, I'll reach out to him and double check #5401

Reply:
This change was already made in the downstream Android projects.

Reply:
Alright, this has been updated!